pom.xml

```
@@ -58,8 +58,8 @@
 	
 	<properties>
 		<!-- metaschema dependencies -->
-		<dependency.metaschema-framework.version>1.0.2</dependency.metaschema-framework.version>
-		<dependency.liboscal-java.version>4.0.2</dependency.liboscal-java.version>
+		<dependency.metaschema-framework.version>1.1.0</dependency.metaschema-framework.version>
+		<dependency.liboscal-java.version>4.1.0-SNAPSHOT</dependency.liboscal-java.version>
```

With the other PR approved don't we pin to the version and not the SNAPSHOT moving forward?

```suggestion
		<dependency.liboscal-java.version>4.1.0</dependency.liboscal-java.version>
```

---

Yes. I needed to release liboscal-java first. Thanks for this.